networks/suzuka/suzuka-full-node/src/partial.rs

```diff
@@ -238,13 +246,34 @@ where
 		Ok(())
 	}
 
+	/// Retries executing a block several times.
+	/// This can be valid behavior if the block timestamps are too tightly clustered for the full node execution.
+	/// However, this has to be deterministic, otherwise nodes will not be able to agree on the block commitment.
+	async fn execute_block_with_retries(
```

Comment:
If the erstwhile `execute_block` is no longer called, remove it and rename this to `execute_block`.
Otherwise this produces dead code that's only going to slow down maintenance.

Reply:
Wdym? This calls `execute_block`?

Reply:
Oh, my bad.